### PR TITLE
Create DDNSS API based on the work of helbgd

### PR DESCRIPTION
**Pull request by mod242**
If you find this request acceptable, I'll edit the Wiki accordingly.

### CODE REVIEW

File: dnsapi/dns_ddnss.sh [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+_ddnss_rest() {
+  method=$1
+  param="$2"
+  _debug param "$param"
+  url="$DDNSS_DNS_API?$param"
+  _debug url "$url"
+
+  # DDNSS uses GET to update domain info
+  if [ "$method" = "GET" ]; then
+    response="$(_get "$url" | sed -e :a -e 's/<[^>]*>//g;/</N;//ba' |tail -n 1 )"
```
Comment (Neilpang) [inline dnsapi/dns_ddnss.sh]: do not use `sed -e`
do not use `tail n`,  please use `_tail_n()` function instead.

**Comment by Neilpang** [comment]
please create an issue for you api, and then edit the wiki page.

Thanks.

File: dnsapi/dns_ddnss.sh (hunk 1 above)
Comment (mod242) [reply dnsapi/dns_ddnss.sh]: Changed the code accordingly, travis check passes as well (even though it seems not to be reflected in the overview here).

**Comment by mod242** [comment]
> please create an issue for you api, and then edit the wiki page.
> 
> Thanks.

Thanks for reviewing. Created the Issue and edited the wiki page.

File: dnsapi/dns_ddnss.sh [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+_ddnss_rest() {
+  method=$1
+  param="$2"
+  _debug param "$param"
+  url="$DDNSS_DNS_API?$param"
+  _debug url "$url"
+
+  # DDNSS uses GET to update domain info
+  if [ "$method" = "GET" ]; then
+    response="$(_get "$url" | sed :a -e 's/<[^>]*>//g;/</N;//ba' | _tail_n 1)"
```
Comment (Neilpang) [inline dnsapi/dns_ddnss.sh]: do not use `sed -e`

Comment (mod242) [reply dnsapi/dns_ddnss.sh]: @Neilpang Just fixed this. I am very sorry that I didn't see this when I first corrected my code. Sorry for the overhead

**Pull request merged**